src/pages/dmt5/_margin_calculator.tsx

```diff
@@ -19,7 +18,7 @@ type CalculatorProps = {
     text: React.ReactElement
     image_name: string
     image_alt_name: string
-    button_text: TString
+    button_text: string
```

Comment:
```sugesstion
    button_text: TString
```
for the translatable strings we need this type, please revert it back.

Reply:
thanks @michio-rework , sorry, not correct resolve conflicts